core-bundle/src/Resources/contao/pages/PageRegular.php

```diff
@@ -699,8 +699,9 @@ protected function createHeaderScripts($objPage, $objLayout)
 	 * Create all footer scripts
 	 *
 	 * @param LayoutModel $objLayout
+	 * @param PageModel $objPage
 	 */
-	protected function createFooterScripts($objLayout)
+	protected function createFooterScripts($objLayout, $objPage = null)
```

Comment:
We could use `global $objPage` instead, then we would not have to change the method signature.

Reply:
It's not used anywhere in this class so I wanted to stick with it. In fact, it should be the same as `createHeaderScripts()` but I cannot just change the method signature so I figured this is the best way.